### PR TITLE
false, logname, time, tty, unclutter: Add Norwegian translation

### PR DESCRIPTION
**Pull request by JoeStone13**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @JoeStone13, welcome and thanks for your submission. Could you please sign the license/cla to pass the check?

File: pages.no/common/false.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# false
+
+> Returnerer en utgangskode på 1.
```
Comment (navarroaxel) [inline pages.no/common/false.md:3]: The verbs should be in `indicative mood`, I don't speak Norwegian but I should check.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
I left 1 comment. Remember *DO NOT* push force commits in the PR please :grin:

**Comment by JoeStone13** [comment]
> 
> 
> Hi @JoeStone13, welcome and thanks for your submission. Could you please sign the license/cla to pass the check?

Hey, sorry about that, it should be signed now.

File: pages.no/common/false.md (hunk 1 above)
Comment (JoeStone13) [reply pages.no/common/false.md]: Noted, I have applied the changes

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @JoeStone13.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by sbrl** [comment]
I don't know what indicative mood is. Can anyone explain at all please? I'm just curious.

**Comment by navarroaxel** [comment]
I'm a native English speaker and maybe I said it in the wrong way, but mebeim said `present indicative` here: https://github.com/tldr-pages/tldr/issues/4714#issuecomment-710824921 and I think he was talking about this: https://en.wikipedia.org/wiki/Realis_mood.

**Comment by navarroaxel** [comment]
@sbrl I'm not a native English speaker and maybe I said it in the wrong way, but mebeim said `present indicative` here: https://github.com/tldr-pages/tldr/issues/4714#issuecomment-710824921 and I think he was talking about this: https://en.wikipedia.org/wiki/Realis_mood.

**Pull request merged**